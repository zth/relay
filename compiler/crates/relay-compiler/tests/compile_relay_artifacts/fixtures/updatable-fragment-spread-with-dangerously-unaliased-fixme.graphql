query updatableFragmentSpreadWithDangerouslyUnaliasedFixmeQuery {
  me {
    ...updatableFragmentSpreadWithDangerouslyUnaliasedFixme_user @dangerously_unaliased_fixme
  }
}

fragment updatableFragmentSpreadWithDangerouslyUnaliasedFixme_user on User @updatable {
  firstName
}
