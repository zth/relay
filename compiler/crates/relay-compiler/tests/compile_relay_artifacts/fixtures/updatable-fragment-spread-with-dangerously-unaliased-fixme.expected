==================================== INPUT ====================================
query updatableFragmentSpreadWithDangerouslyUnaliasedFixmeQuery {
  me {
    ...updatableFragmentSpreadWithDangerouslyUnaliasedFixme_user @dangerously_unaliased_fixme
  }
}

fragment updatableFragmentSpreadWithDangerouslyUnaliasedFixme_user on User @updatable {
  firstName
}
==================================== OUTPUT ===================================
{
  "fragment": {
    "argumentDefinitions": [],
    "kind": "Fragment",
    "metadata": null,
    "name": "updatableFragmentSpreadWithDangerouslyUnaliasedFixmeQuery",
    "selections": [
      {
        "alias": null,
        "args": null,
        "concreteType": "User",
        "kind": "LinkedField",
        "name": "me",
        "plural": false,
        "selections": [
          {
            "args": null,
            "kind": "FragmentSpread",
            "name": "updatableFragmentSpreadWithDangerouslyUnaliasedFixme_user"
          }
        ],
        "storageKey": null
      }
    ],
    "type": "Query",
    "abstractKey": null
  },
  "kind": "Request",
  "operation": {
    "argumentDefinitions": [],
    "kind": "Operation",
    "name": "updatableFragmentSpreadWithDangerouslyUnaliasedFixmeQuery",
    "selections": [
      {
        "alias": null,
        "args": null,
        "concreteType": "User",
        "kind": "LinkedField",
        "name": "me",
        "plural": false,
        "selections": [
          {
            "alias": null,
            "args": null,
            "kind": "ScalarField",
            "name": "__typename",
            "storageKey": null
          },
          {
            "alias": null,
            "args": null,
            "kind": "ScalarField",
            "name": "id",
            "storageKey": null
          }
        ],
        "storageKey": null
      }
    ]
  },
  "params": {
    "cacheID": "9c1625f473e47737ba5ee76d36681bd4",
    "id": null,
    "metadata": {},
    "name": "updatableFragmentSpreadWithDangerouslyUnaliasedFixmeQuery",
    "operationKind": "query",
    "text": null
  }
}

QUERY:

query updatableFragmentSpreadWithDangerouslyUnaliasedFixmeQuery {
  me {
    __typename
    id
  }
}


{
  "argumentDefinitions": [],
  "kind": "Fragment",
  "metadata": null,
  "name": "updatableFragmentSpreadWithDangerouslyUnaliasedFixme_user",
  "selections": [
    {
      "alias": null,
      "args": null,
      "kind": "ScalarField",
      "name": "firstName",
      "storageKey": null
    }
  ],
  "type": "User",
  "abstractKey": null
}
